app/src/main/java/edu/uc/forbesne/shoppingsidekick/MainActivity.kt

```diff
@@ -13,13 +13,12 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.main_activity)
         if (savedInstanceState == null) {
-            supportFragmentManager.beginTransaction()
+                    supportFragmentManager.beginTransaction()
```

Comment:
I fixed the indentation for formatting within the Main Activity. This was recommended in terms of utilizing the space bar and not using tab. (https://kotlinlang.org/docs/coding-conventions.html#class-layout)